app/Http/Controllers/Three/SignupsController.php

```diff
@@ -94,6 +94,12 @@ public function index(Request $request)
             $query = $query->withVisiblePosts();
         }
 
+        if ($request->query('orderByQuantity') === 'desc') {
```

Comment:
Would this make more sense as an `order_by[column]` query parameter (instead of a special case for quantity)?

Reply:
ooo I like that idea @DFurnes ! I know we did this before but not sure how - would we use regex or something to get the column out of the filter?

Reply:
If separating with a comma, you should be able to use `explode`. Something like this:

```php
$orderBy = $request->query('order_by');
if ($orderBy && in_array($orderBy, Signup::$indexes)) {
   list($column, $direction) = explode(',' $orderBy, 2)
   $query = $query->orderBy($column, $direction);
}
```

Reply:
brilliant! Thank you!